sos/plugins/kubernetes.py

```diff
@@ -22,33 +22,75 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    packages = ('kubernetes',)
```

Comment:
Please add 'atomic-openshift' to the packages list so the plugin gets auto-enabled in AP/OSE environments too, otherwise this introduces a regression (you'd have to enable it explicitly when invoking)